doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -86,12 +68,14 @@ In this subsection, gravitational acceleration, and the physical properties of t
 * The ``size distribution type`` parameter specifies the size distribution for each particle type. The acceptable choices are ``uniform`` and ``normal`` distributions.
 
 .. note::
-    For each particle type, two ``size distribution type``s can be defined: ``uniform`` and ``normal``. In ``uniform`` size distribution, the diameter of the particles is constant, while in ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average of ``average diameter`` and standard deviation of ``standard deviation``.
+    For each particle type, three ``size distribution type``s can be defined: ``uniform``, ``normal`` and ``histogram``. In ``uniform`` size distribution, the diameter of the particles is constant. In ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average diameter and a standard deviation of ``standard deviation``.  In the "histogram" size distribution, particle diameters are sampled from a list of diameters and a probability for each diameter value.
```

Comment:
```suggestion
    For each particle type, three ``size distribution type``s can be defined: ``uniform``, ``normal`` and ``histogram``. In ``uniform`` size distribution, the diameter of the particles is constant. In ``normal`` size distribution, the particle diameters are sampled from a normal distribution with an average diameter and a standard deviation of ``standard deviation``.  In the "histogram" size distribution, particle diameters are sampled from a list of diameters and a probability for each diameter value.
```

Can you add an information regarding the fact that this is a probability is number based and not mass based. Ideally I would specify it's a number fraction.